nevergrad/functions/control/core.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import numpy as np
+import typing as tp
+from nevergrad.parametrization import parameter as p
+from ..base import ExperimentFunction
+from .mujoco import GenericMujocoEnv
+
+class BaseFunction(ExperimentFunction):
+    def __init__(self, num_rollouts: int, random_state: tp.Optional[int] = None) -> None:
```

Comment:
how is the `random_state` needed?
I would rather avoid it, because that makes the function deterministic (including in the benchmarks) while it is inherently stochastic.
Then again that may make it a bit harder to set reproducibility correctly. I can do that later

Reply:
``random_state`` is only used to seed generation of *initial state* when ``env.reset()`` thus to control sequence of initial during policy evaluation.